editor/blocks/text-block/index.js

```diff
@@ -1,11 +1,11 @@
-const { query, html } = wp.blocks.query;
+const { html } = wp.blocks.query;
 
 wp.blocks.registerBlock( 'wp/text', {
 	title: 'Text',
 	icon: 'text',
 
 	attributes: {
-		value: query( 'p', html() )
+		value: html()
```

Comment:
This is a good question. If we limit the value to the text within the `p`, we should limit the `Editable` to create a unique `p` and the `onChange` callback of the `Editable` component should extract the content of the `p` before calling the `onChange` prop.

This is probably what we want, I was thinking we could leave this to a separate PR.